Sources/WebSocket/WebSocket.swift

```diff
@@ -90,6 +109,7 @@ public final class WebSocket: BasicWorker {
     ///                 This will be called every time the connected client sends text.
     public func onText(_ callback: @escaping (WebSocket, String) -> ()) {
         onTextCallback = callback
+        self.textBuffer.forEach { onTextCallback(self, $0) }
```

Comment:
After delivering the items to to the callback, we should zero out the array to free memory. It might even be worth making the array an optional so that we can make it `nil` here.

Reply:
I’ll make it an optional